tests/test-data/gen-test-signatures.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
```

Comment:
Let me know what you think of this approach. I wanted to have some simple sanity checks that basic signature validation for all the supported algorithm combinations, and this seemed like a simple way to get that (assuming openssl's output is correct, of course).